### PR TITLE
Cypress Test Base Structure

### PR DESCRIPTION
**Pull request by lenkva**
Fundamental Structure for running Cypress Tests against our website, including an automatic test workflow.

Not yet sure how GitHub will react to automatic workflow calls if an input is not specified - guess we'll find out.

### CODE REVIEW

**Review by steinimo — APPROVED** [review]

**Comment by lenkva** [comment]
Considering that we reviewed this PR together in the meeting earlier today, I'll merge this without waiting for official approval by @steinimo.

**Pull request merged**
